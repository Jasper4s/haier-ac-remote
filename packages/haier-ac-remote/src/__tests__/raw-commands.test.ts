import * as rawCommands from '../lib/raw-commands';

describe('raw-commands', () => {
  it('mac-address', () => {
    const a = rawCommands.macAddress('00:07:A8:17:E9:AC');

    expect(a).toMatchSnapshot();
  });
});
