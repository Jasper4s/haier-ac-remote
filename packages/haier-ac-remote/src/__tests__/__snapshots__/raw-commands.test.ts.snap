// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`raw-commands mac-address 1`] = `"30 30 30 37 41 38 31 37 45 39 41 43 00 00 00 00"`;
